CHANGELOG.md

```diff
@@ -2,6 +2,47 @@
 
 ## Unreleased
 
+## v0.32.0
+
+# 🎉 OpenTelemetry Collector Contrib v0.32.0 (Beta) 🎉
+
+The OpenTelemetry Collector Contrib contains everything in the [opentelemetry-collector release](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.32.0) (be sure to check the release notes here as well!). Check out the [Getting Started Guide](https://opentelemetry.io/docs/collector/getting-started/) for deployment and configuration information.
+
+## 🚀 New components 🚀
+
+- [`cumulativetodelta` processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/cumulativetodeltaprocessor) to convert cumulative sum metrics to cumulative delta
+
+## 💡 Enhancements 💡
+
+- `logzio` exporter:
+  - New implementation of an in-memory queue to store traces, data compression with gzip, and queue configuration options (#4395)
+  - Make `Hclog2ZapLogger` struct and methods private for public go api review (#4431)
+- `newrelic` exporter (#4392):
+  - Marked unsupported metric as permanent error
+  - Force the interval to be valid even if 0
+- `awsxray` exporter: Add PHP stacktrace parsing support (#4454)
+- `file_storage` extension: Implementation of batch storage API (#4145)
+- `alibabacloudlogservice` exporter: Update labels to attributes for OTLP 0.9.0 (#4588)
+- `dynatrace` exporter: Update labels to attributes for OTLP 0.9.0 (#4590)
+- `redis` receiver: Move `interval` runner package to `internal/interval` (#4600)
+- `datadog` exporter:
+  - Skip sum metrics with no aggregation temporality (#4597)
+  - Update labels to attributes for OTLP 0.9.0 (#4589)
+- `elasticsearch` exporter:
+  - Add dedot support (#4579)
+  - Update labels to attributes for OTLP 0.9.0 (#4601)
+- `signalfx` exporter:
+  - Add process metric to translation rules (#4598)
+  - Update labels to attributes for OTLP 0.9.0 (#4603)
+
+## 🧰 Bug fixes 🧰
+
+- `awsxray` exporter: Fix the origin on ECS/EKS/EB on EC2 cases (#4391)
+- `splunk_hec` exporter:
+  - Prevent re-sending logs that were successfully sent (#4467)
+  - Send profiling data separately from logs (#4464)
```

Comment:
This is not a bug fix but rather enhancement. I'd call it "Add support for profiling logs"